tests/jballerina-unit-test/src/test/resources/test-src/expressions/binaryoperations/type-test-expr.bal

```diff
@@ -224,6 +224,28 @@ function testSealedRecordTypes() returns [boolean, boolean] {
     return [a is A3, a is B3];
 }
 
+type Country record {|
+    string code?;
+    string name?;
+    record {|
+        string code?;
+        string name?;
+    |} continent?;
+|};
+
+type MyCountry record {|
+    string code?;
+    record {|
+        string code?;
+    |} continent?;
+|};
+
+function testRecordsWithOptionalFields() {
+    MyCountry x = {};
+    Country y = x;
+    assertTrue(x is Country);
```

Comment:
Shall we use `ballerina/test` for assertion?

Reply:
This `assertTrue()` function was already defined in this `bal` file and used throughout the file instead of `ballerina/test`. That's why I used it here also, do I need to change that?

Reply:
We started to use `ballerina/test` for assertions with the new changes. So, we can gradually remove the usage of those defined functions.

Reply:
I will look into this.